tests/Tests/500__jupyterhub/test-jupyterlab-git.robot

```diff
@@ -0,0 +1,162 @@
+*** Settings ***
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library          OpenShiftCLI
+Library          DebugLibrary
+Suite Setup      Server Setup
+Suite Teardown   End Web Test
+
+
+*** Variables ***
+${REPO_URL} =    *****
+${DIR_NAME} =  Python
+${FILE_PATH} =   Python/file.ipynb
+${EMAIL_ID} =  user@gmail.com
+${NAME} =  user
+${COMMIT_MSG} =  commit msg2
```

Comment:
if you plan to get these at runtime in Jenkins you need to define them in test-variables

Reply:
okay